platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java

```diff
@@ -274,6 +276,38 @@ public boolean getPrefetchesTiles() {
     return nativeMapView.getPrefetchTiles();
```

Comment:
I'm thinking to deprecated these `setPrefetchesTiles` methods to migrate to the the newly added `setPrefetchZoomDelta`, since these two methods are changing the same variable in the native code side. It could be confusing for the user that these method can override each other. Wdyt?

Reply:
Sounds good to me :+1: